src/core/server_status.proto

```diff
@@ -350,9 +350,9 @@ message ServerStatus
   //@@
   HealthRequestStats health_stats = 8;
 
-  //@@  .. cpp:var:: ControlRequestStats control_stats
+  //@@  .. cpp:var:: ModelControlRequestStats control_stats
   //@@
-  //@@     Statistics for Control requests.
+  //@@     Statistics for ModelControl requests.
   //@@
-  ControlRequestStats control_stats = 9;
+  ModelControlRequestStats control_stats = 9;
```

Comment:
Also rename `control_stats`? Is your new API going to add similar thing here?

Reply:
No, I don't think I will need to add Stats for Register/Unregistering Shared Memory Regions.

Reply:
We should have stats for all the APIs. There is no reason to treat register/unregister differently. So control_stats needs to change to model_control_stats.

Reply:
Already done. Will incorporate the stats functionality in my changes for the shared memory control stats